src/views/DashboardView.vue

```diff
@@ -8,11 +8,15 @@
     <template slot="sidebar">
       <h2 class="text-h6 mb-4">Spatial data</h2>
 
-      <layers-list
-        :layers="layers"
-        :initiallySelectedLayers="mangroveLayers"
-        @select-layers="setMangroveLayers"
-      />
+      <div
+        class="dashboard-view__scrollable-list overflow-hidden overflow-y-auto pr-2"
```

Comment:
Only overflow-x should be hidden right? Why not set it with overflow-x-hidden.

Reply:
Actually it isn't even necessary, I have removed it.